tests/_support/Page/AbstractPage.php

```diff
@@ -58,26 +58,51 @@ class AbstractPage
 	 */
 	public static $buttonInstall = 'Install';
 
+	/**
+	 * @var string
+	 */
+	public static $tabTranslations = '//ul[@id="configTabs"]/li[2]/a';
+
 	/**
 	 * @var string
 	 */
 	public static $chooseNo = 'No';
 
+	/**
+	 * @var string
+	 */
```

Comment:
I think xpath is of page redCORE Config. Please move locator of page redCORE Config  to page redCOREConfigPage